HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+  - **HRC** - application-level standards and conventions, including contract standards such as naming and domain standards, name registries, URI schemes, library/package formats, and wallet formats.
+- A **Meta HIP** describes a process surrounding Handshake or proposes a change to (or an event in) a process. Process HIPs are like Standards Track HIPs but apply to areas other than the Handshake protocol itself. They may propose an implementation, but not to Handshake's codebase; they often require community consensus; unlike Informational HIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Handshake  development. Any meta-HIP is also considered a Process HIP.
+- An **Informational HIP** describes an Handshake design issue, or provides general guidelines or information to the Handsahke community, but does not propose a new feature. Informational HIPs do not necessarily represent Handshake community consensus or a recommendation, so users and implementers are free to ignore Informational HIPs or follow their advice.
+
+It is highly recommended that a single HIP contain a single key proposal or new idea. The more focused the HIP, the more successful it tends to be. A change to one client doesn't require an HIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+
+An HIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+
+### Special Requirements for Core HIPs
+
+If a **Core** HIP mentions or proposes changes to the Covenent systems, it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
+```
+TRANSFER (0xfe)
+```
+
+## HIP Work Flow
+
+### Shepherding an HIP
+
+Parties involved in the process are you, the champion or *HIP author*, the [*HIP editors*](#hip-editors), and the [*handshake Client Developers*](#client-developers).
+
+Before you begin writing a formal HIP, you should vet your idea. Ask the Handshake community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is thus recommended to open a discussion thread on [the Handshake Magicians forum] to do this, but you can also use [one of the Handshake Gitter chat rooms], [the Handshake subreddit] or [the Issues section of this repository]. 
```

Comment:
Ok my thought on this since we don't have any of these things, and none of these links work -- maybe HIPs that are just vague ideas should start as issues in this repo, and we can organize discussion there. So if a developer has an idea like "using HNS names for login credentials" but no spec yet, they can open an ISSUE (will review your issue_template as well) with a vague proposal. And then the community can help them move forward or abort if for example, there is already prior art or someone else solved the problem already.